apps/ideas/src/components/modules/IdeaGeneratorButton/AppIdeaGeneratorButton/AppIdeaGeneratorButton.js

```diff
@@ -24,19 +24,45 @@ let fetchIdea = async (modelParameters) => {
     let out = await banana.run(apiKey, modelKey, modelParameters);
     //alert(out);
     //console.log(out);
-    return out.modelOutputs[0].output.split("\n")[0];
-  } catch(e) {
-    return "API Error..."
+    return out.modelOutputs[0].output.split('\n')[0];
+  } catch (e) {
+    return 'API Error...';
   }
-}
+};
 
 //const gamePhraseGenerator = () => <div>{getRandomPhrase(phraseFormats)}</div>;
-const phraseGenerator = async () => <div>{await fetchIdea(modelParameters)}</div>;
+const phraseGenerator = async () => (
+  <div>{await fetchIdea(modelParameters)}</div>
+);
+
+const generateIdeaButtonContent = (
+  <div>
+    <div
+      style={{
+        color: 'white',
+        width: '100%',
+        border: 'solid',
+        borderTop: 0,
+        borderLeft: 0,
+        borderRight: 0,
+      }}
+    >
+      Generate an Idea
+    </div>
+    <div style={{ paddingTop: '9%' }}>
+      Want to generate Ideas for you? Use machine learning to generate an idea
+      to get you started!
+    </div>
+    <div style={{ textAlign: 'right', paddingTop: '1%' }}>
```

Comment:
Could we add a file, `StyledAppIdeageneratorButton.js` for example, to hold these styles instead of using inline styles?